src/handlers/proposalsHandlers.ts

```diff
@@ -43,26 +43,18 @@ const getProposals = (
   next: NextFunction,
 ): void => {
   const paginationParameters = extractPaginationParameters(req);
-  db.sql(
-    'proposals.selectWithPagination',
-    {
-      ...getLimitValues(paginationParameters),
-    },
-  )
-    .then((proposalsQueryResult: Result<Proposal>) => {
-      logger.debug(proposalsQueryResult);
-      const { rows: proposals } = proposalsQueryResult;
-      if (isProposalArray(proposals)) {
-        res.status(200)
-          .contentType('application/json')
-          .send(proposals);
-      } else {
-        next(new InternalValidationError(
-          'The database responded with an unexpected format.',
-          isProposalArray.errors ?? [],
-        ));
-      }
-    })
+  (async () => {
+    const proposalBundle = await loadBundle<Proposal>(
```

Comment:
4

I like how the use of generics turned out. We can grab a `Bundle<Proposal>`. That is nice and satisfying.

As for the term "load", I think I have seen "load" mean "write" and I have seen "load" mean "read." So I would rather it be "get" or "select" or "read" for reads and "post" or "insert" or "write" for writes. This comment about "load" applies to several more spots, not only here.

Reply:
`load` meaning "write" is very surprising! Where did you see that, @bickelj?

Reply:
FWIW I'm comfortable using `get` instead of `load`

I think of get as an http verb and load as a database verb, but that distinction is probably invented and might not matter.

Reply:
Yeah, `get` does have an http ring, and it would feel weird to `post` to the database. However, `load` and `store` work fine. And `read` and `write` would work, though technically all these include both reading from one place and writing to another. :fist_oncoming: :skull_and_crossbones: :horse:

Reply:
Further beating the dead horse: what does the "L" in ETL mean?

Reply:
<img width="765" alt="image" src="https://user-images.githubusercontent.com/208884/233103327-ec562079-cc49-4337-abc7-22cb62fc3f39.png">
<img width="765" alt="image" src="https://user-images.githubusercontent.com/208884/233103375-615c6311-d268-4210-87a7-bd571bcf7d6f.png">